include/glow/Backends/QueueBackedDeviceManager.h

```diff
@@ -32,7 +32,7 @@ class QueueBackedDeviceManager : public DeviceManager {
   std::atomic<runtime::RunIdentifierTy> nextIdentifier_{1};
 
 public:
-  QueueBackedDeviceManager(BackendKind backend);
+  QueueBackedDeviceManager(BackendKind backend, std::string name);
```

Comment:
Should this be stringRef?

Reply:
At a certain point we want to store a copy, but we can do that at the last stage.